### PR TITLE
[Arista] Add tuning values for phys on 7280cr3mk

### PR DESCRIPTION
**Pull request by byu343**
#### Why I did it
This change specifies the tuning values for each lane of the B52 phy chips. These values can be different for different ports. The values being set are under the assumption of optical transceivers. This change depends on the change to sonic-swss: https://github.com/Azure/sonic-swss/pull/2158.

#### How to verify it
We verified the values are correctly set on the B52 chips of Arista 7280cr3, by reading them from the debug cli of the B52 driver.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Comment by sujinmkang** [comment]
Do we have any dependency between the two prs?

**Comment by byu343** [comment]
> Do we have any dependency between the two prs?

Yes, the values added here can take effect only after the sonic-swss PR (https://github.com/Azure/sonic-swss/pull/2158) is merged.

File: device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json [hunk 1]
```diff
@@ -119,224 +119,544 @@
...
+      "system_tx_fir_pre2": [1,1],
+      "system_tx_fir_pre1": [-5,-5],
+      "system_tx_fir_main": [14,14],
+      "system_tx_fir_post1": [0,0],
+      "system_tx_fir_post2": [0,0],
+      "line_tx_fir_pre2": [0,0,0,0],
+      "line_tx_fir_pre1": [-1,-1,-1,-1],
+      "line_tx_fir_main": [13,13,13,13],
+      "line_tx_fir_post1": [-5,-5,-5,-5],
+      "line_tx_fir_post2": [0,0,0,0]
```
Comment (prgeor) [inline device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json:251]: The line side SI settings may depend upon the type of optics for some platforms. In that case, Xcvrd should read these values based upon the media ([see Ref](https://github.com/Azure/sonic-platform-daemons/blob/master/sonic-xcvrd/xcvrd/xcvrd.py#L639)) and publish to gearbox syncd to program the line side on the gearbox. Can we discuss this scheme in the community?

**Comment by kenneth-arista** [comment]
Tagging @arlakshm for awareness

**Comment by kenneth-arista** [comment]
@arlakshm gentle reminder to merge this change when you get a chance

**Comment by arlakshm** [comment]
Hi @prgeor, can you help approve and merge this PR

**Comment by kenneth-arista** [comment]
Another gentle reminder to consider merging. All checks have passed. This PR has been open for nearly 6 months.

**Comment by kenneth-arista** [comment]
@arlakshm and @prgeor a reminder to review this. Thanks!

File: device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json (hunk 1 above)
Comment (byu343) [reply device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json:251]: @prgeor , this gearbox_config.json here is just to provide some initial value like setting tuning values for brcm ASIC in brcm config file. For the dynamic tuning support (based on xcvr type), an idea is to add the support in the future with some extension to the usage of media_settings.json.

**Comment by byu343** [comment]
@prgeor @sujinmkang The sonic-swss change (https://github.com/sonic-net/sonic-swss/pull/2158) that this change depends on has been merged. Can you please help to review this change again? Thanks.

**Review by prgeor — APPROVED** [review]

**Comment by prgeor** [comment]
@lguohan please help merge

**Pull request merged**